.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I don't use global gitignore like many other people. It's okay for npmdebug or another single file. But node_modules is too big for git status and should be in every package. It's my opinion.